tidalapi/playlist.py

```diff
@@ -83,7 +90,8 @@ def parse(self, json_obj):
         self.created = dateutil.parser.isoparse(created) if created else None
         public = json_obj.get("publicPlaylist")
         self.public = None if public is None else bool(public)
-        self.popularity = json_obj.get("popularity")
+        popularity = json_obj.get("popularity")
+        self.popularity = int(popularity) if popularity else None
```

Comment:
This breaks the tests, as we expect default value to be 0. However, your changes are more consistent so lets update the tests to expect None